sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/meta/store/InMemoryMetaStore.java

```diff
@@ -37,7 +37,8 @@
   private Map<String, Table> tables = new HashMap<>();
   private Map<String, TableProvider> providers = new HashMap<>();
 
-  public InMemoryMetaStore() {
+  @Override public String getTableType() {
+    return "";
```

Comment:
I think it should have its own table type still

Reply:
Ok, I added type of `store`.